scripts/gen_csr_test.py

```diff
@@ -312,6 +321,12 @@ def gen_csr_instr(csr_map, csr_instructions, xlen, iterations, out):
         help="Specify how many tests to be generated")
 parser.add_argument("--out", type=str, default="./",
         help="Specify output directory")
+parser.add_argument("--pass_val", type=int, default=0,
```

Comment:
Use the same value as the run script? Actually I feel that there's no need to add this argument. PASS = 1, FAIL = 0 seems to be good enough.  This could simplify the script interface.

Reply:
Sure